SokolTextGame/Program.cs

```diff
@@ -1,6 +1,34 @@
 ﻿using SokolTextGame;
+using System.Drawing;
```

Comment:
This probably got added automatically by Visual Studio. You don't need System.Drawing, remove it please.